### ISSUE TITLE
Django: Consider moving templates into relecloud folder

### ISSUE THREAD

**Issue by pamelafox** [kjaymiller/cookiecutter-relecloud#215]
Right now our views have to explicitly provide the template name, like for the detail views, and thats because we haven't nested them in a relecloud folder. If we did nest them in a relecloud folder, then more of the django auto-guessing templates logic would work. So maybe we should consider that. Probably in postgen hooks since templates is a shared dir.

**Comment by kjaymiller** [comment]
Yeah I did the current setup before we had as many _post_gen_ content. I do think that we can probably move that templatest folder although I don't think it's that out of the ordinary for there to be a templates path that can be shared across apps.
